frontend/src/components/pages/rp-connect/Pipelines.Create.tsx

```diff
@@ -84,10 +83,24 @@ class RpConnectPipelinesCreate extends PageComponent<{}> {
     const alreadyExists = pipelinesApi.pipelines.any((x) => x.id === this.fileName);
     const isNameEmpty = this.fileName.trim().length === 0;
 
+    const CreateButton = () => {
+      const toast = useToast();
```

Comment:
Do we need this component? Can't we stick to `createStandaloneToast` approach? I don't like that we create these components in the render function, that doesn't look right. I think using `onSuccess`/`onError` callbacks and trying to avoid abstraction above `<Button` would be more standard.

Reply:
I suggested to use `showToast` in a Slack message, as we don't want to always define a new toast provider, @jvorcak are there any common errors we may face if we don't provide a standalone toast?

Reply:
Yeah, the issue was with page change hiding the toast

Reply:
@malinskibeniamin it should be fine. The only reason I wrote this is that I'm not a huge fan of this interface definition (like I wrote above). But it's definitely not a blocker.